salesforce-gui/src/test/resources/features/account/search_account.feature

```diff
@@ -5,8 +5,11 @@ Feature: Search Account
   Scenario: Account search using the search tool and compare with the account created
     Given I log in to Salesforce with Account Owner User credentials
     When I go to "HOME" page
-    And I search the following name
-      | Armando Guerra |
+    And I search the following values <search> and compare the result with the <filePath> file
+      | Search        | Type of search | filePath                     |
+      | Armando Lopez | Account        | searchByName.json            |
+      | Lopez         | Account        | searchByLastName.json        |
+      | Ar            | Account        | searchByFirstCharacters.json |
     Then I search the account name in the drop down menu
```

Comment:
If you remember, Yesica said that verify only searching the account in the dropdown menu is insufficient; please add a step "AND the gotten results should match with the inserted text to search" in the line 14.